src/mcts/search.cc

```diff
@@ -1542,12 +1542,9 @@ void SearchWorker::DoBackupUpdateSingleNode(
       d = n->GetD();
       m = n->GetM();
     }
-    n->FinalizeScoreUpdate(v / (1.0f + params_.GetShortSightedness() * depth),
```

Comment:
Looks like this mostly reverts https://github.com/LeelaChessZero/lc0/commit/c0705b77faa20b042ed8d24a57f63c36f46f80c4 although that commit did add `depth` which is now unused but the optimizer knows to remove it anyway.

I believe the only other code interaction with short sightedness was to account for adjusted terminal win/loss values, e.g., wl > 0 vs wl == 1.

Reply:
Depth removed.